spell/spell-reviewer-goerli-checklist.md

```diff
@@ -23,7 +23,7 @@ Spell Actions:
   * [ ] check on-chain interface of deployed contract via `cast interfaces <contract_address>` to ensure correctness
   * [ ] interface naming style should match with `Like` suffix (e.g. `VatLike`)
   * [ ] ensure they only list used functions in spell code
-* [ ] Rates Match
+* [ ] Rates matches
```

Comment:
Technically it is "Rates match" so this old change was OK

Reply:
18cf91c